packages/@react-spectrum/s2/src/Accordion.tsx

```diff
@@ -0,0 +1,368 @@
+        className={(UNSAFE_className ?? '') + itemStyles}>
+        {props.children}
+      </RACAccordionItem>
+      {!isQuiet && <Divider UNSAFE_className={lastDividerStyles} size="S" />}
+    </>
+  );
+}
+
+/**
+ * A accordion item is a collapsible section of content. It is composed of a a header with a heading and trigger button, and a panel that contains the content.
+ */
+let _AccordionItem = /*#__PURE__*/ (forwardRef as forwardRefType)(AccordionItem);
+export {_AccordionItem as AccordionItem};
+
+export interface AccordionHeaderProps extends UnsafeStyles, DOMProps {
+  children: React.ReactNode
+}
+
+const headingStyle = style({
+  margin: 0
+});
+
+const buttonStyles = style({
+  // Custom focus ring so it doesn't overlap divider
+  outlineStyle: 'solid',
+  outlineColor: {
+    default: 'transparent',
+    isHovered: 'transparent-black-100',
+    isFocusVisible: 'focus-ring'
+  },
+  outlineWidth: 2,
+  outlineOffset: 0,
+  font: 'heading',
+  color: {
+    default: 'neutral',
+    isDisabled: 'disabled'
+  },
+  fontWeight: 'bold',
+  lineHeight: 'ui',
+  display: 'flex',
+  alignItems: 'center',
+  paddingX: {
+    size: {
+      S: sizeValue(7),
+      M: sizeValue(11),
+      L: sizeValue(14),
+      XL: sizeValue(17)
+    }
+  },
+  paddingY: {
+    size: {
+      S: {
+        density: {
+          compact: sizeValue(2),
+          regular: sizeValue(5),
+          spacious: sizeValue(9)
+        }
+      },
+      M: {
+        density: {
+          compact: 4,
+          regular: 8,
+          spacious: 12
+        }
+      },
+      L: {
+        density: {
+          compact: 4,
+          regular: sizeValue(9),
+          spacious: 12
+        }
+      },
+      XL: {
+        density: {
+          compact: sizeValue(5),
+          regular: sizeValue(9),
+          spacious: sizeValue(13)
+        }
+      }
+    }
+  },
+  gap: {
+    size: {
+      S: sizeValue(7),
+      M: sizeValue(11),
+      L: sizeValue(14),
+      XL: sizeValue(17)
+    }
+  },
+  height: '[inherit]',
+  minHeight: 'control',
+  width: 'full',
+  backgroundColor: {
+    default: 'transparent',
+    isFocusVisible: 'transparent-black-100',
+    isHovered: 'transparent-black-100'
+  },
+  borderWidth: 0,
+  borderRadius: {
+    // Only rounded for keyboard focus and quiet hover.
+    default: 'none',
+    size: {
+      S: {
+        isFocusVisible: '[7px]',
+        isQuiet: {
+          isHovered: '[7px]',
+          isFocusVisible: '[7px]'
+        }
+      },
+      M: {
+        isFocusVisible: 'default',
+        isQuiet: {
+          isHovered: 'default',
+          isFocusVisible: 'default'
+        }
+      },
+      L: {
+        isFocusVisible: '[9px]',
+        isQuiet: {
+          isHovered: '[9px]',
+          isFocusVisible: '[9px]'
+        }
+      },
+      XL: {
+        isFocusVisible: 'lg',
+        isQuiet: {
+          isHovered: 'lg',
+          isFocusVisible: 'lg'
+        }
+      }
+    }
+  },
+  textAlign: 'start'
```

Comment:
should be the default

Reply:
This is to override the user agent stylesheet's default button styles of `text-align: center`